projects/js-packages/charts/src/components/shared/with-responsive.tsx

```diff
@@ -32,14 +32,13 @@ export function withResponsive< T extends BaseChartProps< unknown > >(
 
 		// Calculate dimensions
 		const containerWidth = parentWidth ? Math.min( parentWidth, maxWidth ) : 600;
-		const containerHeight = containerWidth * aspectRatio;
+		const containerHeight = props.height ?? containerWidth * aspectRatio;
 
 		return (
 			<div
 				ref={ parentRef }
 				style={ {
 					width: '100%',
-					minHeight: `${ containerHeight }px`,
```

Comment:
It otherwise set a min height for the wrapper which creates a big gap when the height is specificed.